src/ucs/sys/event_set.h

```diff
@@ -15,12 +15,18 @@
  */
 typedef struct ucs_sys_event_set ucs_sys_event_set_t;
 
+/**
+ * ucs_event_set_extra_args used in ucs_event_set_wait callback function.
+ *
+ */
+typedef void ucs_event_set_extra_args;
```

Comment:
Should I define `void *` or `void **`? If so Could you tell me how to create new data type `void **` or `void *`.

Reply:
as the name of this type is `*_args`, then we can define it as `void**`, i.e.
```
typedef void** ucs_event_set_extra_args;
```

Reply:
Do you prefer `typedef void** ucs_event_set_extra_args;`?

If so, could you tell me the usage of the following?
How to use `foo` variable?

```C
#include <stdio.h>
typedef void** ucs_event_set_extra_args;

int main(int argc,char *argv[]){
  int test = 1;
  ucs_event_set_extra_args foo;
  void *bar[2];

  bar[0] = &test;

}
```

Reply:
I've got your example.

so, we can define:
`typedef void* ucs_event_set_extra_arg;`  it define sthe type for argument

Then it could be used as:
```
#include <stdio.h>
typedef void* ucs_event_set_extra_args;

int main(int argc,char *argv[]){
  int test = 1;
  void *bar[2];
  ucs_event_set_extra_arg foo[2];

  bar[0] = &test;
  bar[1] = &test;

  foo[0] = &test;
  foo[1] = &bar;

  /* nwo we can use foo */
}
```

Reply:
I got it. I'll modify to `typedef void* ucs_event_set_extra_args;`

Reply:
> I got it. I'll modify to `typedef void* ucs_event_set_extra_args;`

please, not this should be:
`typedef void* ucs_event_set_extra_arg;` because it is for *argument* (not for *argumentS*)

for arguments we will use an array of `ucs_event_set_extra_arg`, i.e. `ucs_event_set_extra_arg args[N];`

Reply:
Thanks. Use `typedef void* ucs_event_set_extra_arg;` not **argumentS**